### PR TITLE
Add Argentinian CBU local flavor

### PR DESCRIPTION
**Pull request by agustin380**
This pull request adds a custom field for Clave Bancaria Uniforme, used in all financial transactions in Argentina.

More info (in spanish):
http://www.clientebancario.gov.ar/default.asp
http://es.wikipedia.org/wiki/Clave_Bancaria_Uniforme

### CODE REVIEW

**Comment by agustin380** [comment]
@benkonrath Is there any problem with this pull request? It's been some time since I pushed it...

File: localflavor/ar/forms.py [hunk 1]
```diff
@@ -137,3 +137,62 @@ def _format(self, cuit, check_digit=None):
...
+    The remaining 14 digits denote an account number, plus a verifying digit.
+
+    More info:
+    http://www.clientebancario.gov.ar/default.asp
+    """
+    default_error_messages = {
+        'invalid': _('Enter a valid CBU in XXXXXXXXXXXXXXXXXXXXXX format.'),
+        'max_length': _('CBU must be exactly 22 digits long.'),
+        'min_length': _('CBU must be exactly 22 digits long.'),
+        'checksum': _("Invalid CBU."),
```
Comment (benkonrath) [inline localflavor/ar/forms.py]: Minor point: Can you change the double-quotes to single quotes for consistency.

File: localflavor/ar/forms.py [hunk 2]
```diff
@@ -137,3 +137,62 @@ def _format(self, cuit, check_digit=None):
...
+
+
+class ARCBUField(CharField):
+    """
+    This field validates a CBU (Clave Bancaria Uniforme). A CBU is a 22-digits long
+    number. The first 8 digits denote bank and branch number, plus a verifying digit.
+    The remaining 14 digits denote an account number, plus a verifying digit.
+
+    More info:
+    http://www.clientebancario.gov.ar/default.asp
```
Comment (benkonrath) [inline localflavor/ar/forms.py]: Can you also add the wikipedia entry? Thanks.
https://es.wikipedia.org/wiki/Clave_Bancaria_Uniforme

Comment (benkonrath) [inline localflavor/ar/forms.py]: The comment should also have `.. versionadded:: 1.3` at the bottom.

**Comment by benkonrath** [comment]
@agustin380 Sorry for the delay. Thanks for the reminder. It looks good. Besides those minor points, could you also add a changelog entry. Please rebase to master and squash any changes you make into one commit. Thanks!

**Comment by agustin380** [comment]
@benkonrath Done, thanks for the feedback!

File: docs/changelog.rst [hunk 3]
```diff
@@ -15,6 +15,8 @@ New flavors:
 
 New fields for existing flavors:
 
+- Added ARCBUField model field.
```
Comment (benkonrath) [inline docs/changelog.rst]: You actually added a form field, not a model field. :) Can you change this? Thanks.

Comment (agustin380) [inline docs/changelog.rst]: @benkonrath Damn, missed that one. It's fixed now, I hope I didn't miss anything else. Thanks again, I'll be making another PR on the following days, please stay around! ;P

**Pull request merged**